test/fixtures/suspay.json

```diff
@@ -0,0 +1,59 @@
+{
+  "SUSPAY_CREATE_META_JSON": {
+    "TransactionIndex": 2,
+    "AffectedNodes": [
+      {
+      "CreatedNode": {
+        "LedgerEntryType": "SusPay",
```

Comment:
It is from rippled: https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/impl/LedgerFormats.cpp#L84